src/widgets/spinbox/lv_spinbox.c

```diff
@@ -354,7 +354,9 @@ static void lv_spinbox_constructor(const lv_obj_class_t * class_p, lv_obj_t * ob
     lv_textarea_set_one_line(obj, true);
     lv_textarea_set_cursor_click_pos(obj, true);
 
+#if LV_WIDGETS_HAS_DEAFULT_VALUE
     lv_spinbox_updatevalue(obj);
```

Comment:
As we need to set some defaults anyway, here I would keep it for consistency to apply the previously set values.